librz/core/cmd_type.c

```diff
@@ -454,6 +454,89 @@ static void types_function_print_all(RzCore *core, RzOutputMode mode) {
 	}
 }
 
+static void types_link_print(RzCore *core, const char *type, ut64 addr, RzOutputMode mode) {
+	rz_return_if_fail(type);
+	switch (mode) {
+	case RZ_OUTPUT_MODE_JSON: {
+		PJ *pj = pj_new();
+		if (!pj) {
+			return;
+		}
+		pj_o(pj);
+		char *saddr = rz_str_newf("0x%" PFMT64x, addr);
```

Comment:
why this instead of just using `pj_kn`?

Reply:
It uses this value as a key, not a value. We don't have a PJ API for numerical keys. We probably should.